miniupnpd/natpmp.c

```diff
@@ -299,7 +299,7 @@ void ProcessIncomingNATPMPPacket(int s, unsigned char *msg_buff, int len,
 				unsigned short eport2, iport2;
 				char iaddr2[16];
 				int proto2;
-				char desc[64];
+				char desc[1016];
```

Comment:
there is no need for 1016 to store the NAT-PMP specific fake description. Even 64 is too much :)